resources/views/latex/import.blade.php

```diff
@@ -38,7 +38,7 @@
 
 \begin{enumerate}
 @foreach($items as $item)
-    \item {{ $item }}
+    \item {{ $item->name }} ({{ $item->serial_number }})
```

Comment:
should be displayed only if the item has a serial number (currently `( )` shown in this case)